### PR TITLE
improve template.env documentation; upgrade to postgres v12

### PR DESCRIPTION
**Pull request by Niicck**
This is good!
- [ ] We're upgraded to postgres v12
- [ ] We're more secure (local postgres now has a password)
- [ ] The template.env file is up to date with better documentation for onboarding
- [ ] There's a helper script to transform your .env values into a pycharm digestible form
- [ ] removed lingering V3_WIP zombie code

### CODE REVIEW

File: scripts/make_pycharm_vars.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+from os import path
+env = open(path.join(path.dirname(__file__), '../.env'), "r")
+
+pycharm_vars = ""
+for line in env.readlines():
+    if not line.startswith("#"):
+        pycharm_vars = pycharm_vars + line.strip() + ";"
+
+print(pycharm_vars)
```
Comment (chiaberry) [inline scripts/make_pycharm_vars.py:12]: yaay

File: .circleci/vars/heroku_pr_vars.py [hunk 2]
```diff
@@ -66,7 +66,6 @@
     "DELETION_PROTECTION": 0,
     "DJANGO_LOG_LEVEL": "INFO",
     "LOAD_DATA": "fixtures",
-    "V3_WIP": True,
```
Comment (briaguya) [inline .circleci/vars/heroku_pr_vars.py:69]: hooray for getting rid of unused env vars!

File: template.env [hunk 3]
```diff
@@ -1,8 +1,16 @@
...
+DATABASE_URL=postgres://joplin:${JOPLIN_POSTGRES_PASSWORD}@127.0.0.1:5433/joplin
+
+# Django vars
+DEBUG=1
+DEBUG_TOOLBAR=0
 MONITOR_PERFORMANCE=False
-DEPLOYMENT_MODE=LOCAL
-DJANGO_LOG_LEVEL=INFO
+DJANGO_LOG_LEVEL=DEBUG
 DJANGO_SECRET_KEY=dummy
```
Comment (briaguya) [inline template.env:10]: do we know what it would take to change this to 'choose_your_own' as well? Would it break things?

Comment (Niicck) [reply template.env:10]: Nope, you can change DJANGO_LOG_LEVEL, it wouldn't break anything. 
`template.env` is just a template for your own local `.env` file. You can set whichever values you'd prefer in your .env.

Comment (Niicck) [reply template.env:10]: But you're right. DJANGO_LOG_LEVEL=INFO is a better default value.

Comment (Niicck) [reply template.env:10]: Oh, you meant DJANGO_SECRET_KEY. Yeah, should be a choose_your_own.

**Pull request merged**